pandas/core/generic.py

```diff
@@ -2530,6 +2535,26 @@ def to_json(
             date_format = "iso"
         elif date_format is None:
             date_format = "epoch"
+            dtypes = (
+                self.dtypes.values
+                if self.ndim == 2
+                else np.array([self.dtype], dtype=object)
+            )
+            if any(lib.is_np_dtype(dtype, "mM") for dtype in dtypes):
+                warnings.warn(
+                    "The default 'Epoch' date format is deprecated and will be removed "
+                    "in a future version, please use 'iso' date format instead.",
+                    FutureWarning,
+                    stacklevel=find_stack_level(),
+                )
+        elif date_format == "epoch":
+            # GH#57063
+            warnings.warn(
+                "'Epoch' date format is deprecated and will be removed in a future "
```

Comment:
```suggestion
                "'epoch' date format is deprecated and will be removed in a future "
```

Very minor but let's keep captialization consistent